lua/DCS-gRPC/methods/mission.lua

```diff
@@ -411,3 +413,59 @@ GRPC.onDcsEvent = function(event)
     return nil
   end
 end
+
+function missionCommandCallback(callbackId)
```

Comment:
Is the intention here being that DCS-gRPC clients will recieve notifications when a radio command (added via thse APIs) has been triggered by a player?

If so then I don't think this is how it works but I will let @rkusa weigh in on this since he is the expert on the streaming side of things.

Reply:
I would have added this as a custom event, too. So it is good that way IMO

Reply:
ah yeah no need for the `GRPC.success` here